src/index.js

```diff
@@ -0,0 +1,7 @@
+/* eslint-disable no-unused-vars */
+import css from './style.css';
+import getData from './getData';
```

Comment:
are these lines needed? :thinking:

Reply:
They were there simply for testing I forgot to remove them thank you.